src/mocks/data/draft_reports_object.json

```diff
@@ -0,0 +1,50 @@
+{
+  "status": "processing", // enum: one of ["processing", "failed", "delivered"]
+  "current_stage": "dex-upload",
+  "current_action": "metadata-validation",
+  "file_name": "test.txt",
+  "transport_id": "unique_guid", //represents the id for the entire transaction i.e. upload_id
+  "timestamp": "12-12-2023", -- first initial timestamp
+  "data_stream_id": "aims-celr",
+  "data_stream_route": "csv",
+  "reports": [
+    {
+    "stage": "DEX HL7 v2", //describes a chunk of the pipeline, usually associated with a specific product; string could be human readable, or transformed from id to human readable
+    "action": "debatch", //describes specific thing that happened as part of the stage
+    "schema_name": "dex-hl7",
+    "version": "1.0",
+    "data_stream_id": "aims-celr",
+    "data_stream_route": "csv",
+    "jurisdiction": "MD",
+    "transport_id": "unique_guid", //represents the id for the entire transaction i.e. upload_id
+    "parent_id": "unique_guid", (would be null if it is the parent?) //id for the thing that resulted in this report?
+    "message_id": "unique_guid",//id for the thing this report is about
+    "type": "failure", //represents machine readable way to understand what happened in the stage action; enum: one of ["failure", "success", "pending"] ("delivered?")
```

Comment:
type? this sounds more like status!

Reply:
It is the type of report, not the overall `status` of the upload. I agree this naming could be confusing. We could use the word `status` at both levels. The report level, and the higher level of the overall submission status. @fearless-abbey how do you feel about using `status` in both places?